docs/data-sources/builtin-toolsets/index.md

```
@@ -2,47 +2,89 @@
 
 HolmesGPT includes pre-built integrations for popular monitoring and observability tools. Some work automatically with Kubernetes, while others require API keys or configuration.
 
-## Available Toolsets
+### Cloud Providers
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)